adam/visualization/make_scenes.py

```diff
@@ -3,11 +3,17 @@
    operating and when other code (gathering and processing scene information)
    is executing in a serial manner.
    """
-from typing import Iterable, List, Tuple, Union, DefaultDict
+from typing import Iterable, List, Tuple, Union, DefaultDict, Optional, Callable, Any
```

Comment:
If this module is not yet using the positioning stuff, can we shift it to another PR that I can review later?